types/leaflet/index.d.ts

```diff
@@ -241,33 +334,99 @@ export abstract class Evented extends Class {
      * keyword will point to). You can also pass several space-separated types
      * (e.g. 'click dblclick').
      */
+    // tslint:disable:unified-signatures
```

Comment:
Hm not sure how I feel about this to be honest

Reply:
@Favna unfortunately, it's necessary. Otherwise it pushes to merge the `type, fn, ctx?` overloads with the `eventMap` overload, which would be inaccurate because `eventMap` overloads do not take optional `ctx`.